### PR TITLE
Desktop upgrade workaround

### PR DESCRIPTION
**Pull request by usha-mandya**
Adding the known issue and workaround for the Desktop upgrade issue.

### CODE REVIEW

File: docker-for-windows/release-notes.md [hunk 1]
```diff
@@ -82,6 +82,27 @@ Docker Desktop Community 2.1.0.0 contains the following experimental features:
 * When two services have a common exposed port, Docker Desktop exposes the available ports for the second service.
 * Fixed the UI lock when changing the Kubernetes state.
 
+### Known issue
```
Comment (mikeparker) [inline docker-for-windows/release-notes.md]: Make it plural? `issues`?

Comment (usha-mandya) [reply docker-for-windows/release-notes.md]: Made it singular as we only have one known issue. I think it reads better with the plural. I've updated the heading.

File: docker-for-windows/release-notes.md [hunk 2]
```diff
@@ -82,6 +82,27 @@ Docker Desktop Community 2.1.0.0 contains the following experimental features:
 * When two services have a common exposed port, Docker Desktop exposes the available ports for the second service.
 * Fixed the UI lock when changing the Kubernetes state.
 
+### Known issues
+
+Attempts to upgrade Docker Desktop to version 2.1.0.0 can fail with the error `value cannot be null`. To work around this issue:
+
+1. Open the `settings.json` file located at `%APPDATA%\Docker`.
```
Comment (gtardif) [inline docker-for-windows/release-notes.md]: I would be more precise here with
%APPDATA%\Docker\settings.json (e.g. C:\Users\USERNAME\AppData\Roaming\Docker\settings.json)

Comment (usha-mandya) [reply docker-for-windows/release-notes.md]: Thanks. I've updated the location.

**Comment by usha-mandya** [comment]
@gtardif - The precise location of the settings.json file has been added.

**Pull request merged**